debug-init-env.sh

```diff
@@ -17,6 +17,14 @@
 
 echo -n "Pre-creating log and database..."
 TEMPDIR=data
-sqlite3 $TEMPDIR/database.db < src/test/resources/database/database.sql
+
+cp src/test/resources/database/database.sql home/install/db-tables.sql
+
+mkdir data/database
+echo 'urlid pidgen' > data/database/db.rc
+echo 'url jdbc:hsqldb:file:data/database/db;shutdown=true' >> data/database/db.rc
+
+java -jar lib/sqltool.jar --rcFile=data/database/db.rc pidgen home/install/db-tables.sql
```

Comment:
Your comment inspired me to change it all together. I saw that the same files were generated every time by `sqltool.jar`, so I decided to put these generated files in `src/test/resources/database`, next to `database.sql`. These now get copied to `data/database` when running `run-reset-env.sh`.

To avoid having to edit a generated file when `database.sql` changes, I included a new script that generates these files for you. It does the downloading of `sqltool.jar` and `hsqldb.jar`, runs the command and does cleanup afterwards. Afterwards you can commit the generated files (read: their diffs) to version control.

I also included a new `README.md` file in `src/test/resources/database` to explain the files there and to give instructions on how to run that script.